libs/config_engine.py

```diff
@@ -196,7 +197,7 @@ def get_areas(self):
 
     def should_send_email_notifications(self, entity):
         if "emails" in entity:
-            if os.path.isfile("oauth2_cred.json"):
+            if is_email_configured():
```

Comment:
I think that it should be:

```suggestion
            if is_mailing_configured():
```